internal/bminventory/inventory.go

```diff
@@ -304,6 +306,35 @@ func (b *bareMetalInventory) updatePullSecret(pullSecret string, log logrus.Fiel
 	return pullSecret, nil
 }
 
+func getDefaultHighAvailabilityAndMasterCountParams(highAvailabilityMode *string, controlPlaneCount *int64) (*string, *int64) {
```

Comment:
I think this should be a common check we we can use it in other places like operators / feature support etc..

Reply:
Do you mean the validation func ?